handle_balance_request.go

```diff
@@ -34,7 +34,9 @@ func (svc *Service) HandleGetBalanceEvent(ctx context.Context, request *Nip47Req
 			"appId":     app.ID,
 		}).Errorf("App does not have permission: %s %s", code, message)
 
-		return svc.createResponse(event, Nip47Response{Error: &Nip47Error{
+		return svc.createResponse(event, Nip47Response{
+			ResultType: NIP_47_GET_BALANCE_METHOD,
```

Comment:
I am wondering, we could use `request.Method` in all cases which may decrease chance of returning the wrong method by a careless copy-paste